### PR TITLE
Add `Vocaloid Mapping Contest 2 Results` news post

### PR DESCRIPTION
**Pull request by osuWhite**

### CODE REVIEW

File: wiki/Contests/VMC/2/en.md [hunk 1]
```diff
@@ -88,6 +88,50 @@ Please be sure to support [iroha(sasaki)](https://osu.ppy.sh/beatmaps/artists/39
 | 14 (Miku ver.) | Hatsune Miku | π | 4:55 | 128 | [.osz](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.osz) |
 | sigh | Kagamine Rin | iroha(sasaki) | 4:08 | 148 | [.osz](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.osz) |
 
+## Results 
+
+### osu!
+
+Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91885b2923e167f79fd44)!
```
Comment (Walavouchey) [inline wiki/Contests/VMC/2/en.md:95]: the uses a more encyclopedic tone

```suggestion
Detailed scores, judge comments, and final placements are [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91885b2923e167f79fd44).
```

File: wiki/Contests/VMC/2/en.md [hunk 2]
```diff
@@ -88,6 +88,50 @@ Please be sure to support [iroha(sasaki)](https://osu.ppy.sh/beatmaps/artists/39
...
+| :-: | :-- | :-- | :-- |
+| 1st | [Nuolong](https://osu.ppy.sh/users/4732455) | world language | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
+| 2nd | [yaspo](https://osu.ppy.sh/users/4945926) | how prove | - |
+| 3rd | [Meijiro McQueen](https://osu.ppy.sh/users/11555612) | having mental | - |
+| 4th | [celerih](https://osu.ppy.sh/users/4696296) | president summer | - |
+| 5th | [Nijika Ijichi](https://osu.ppy.sh/users/10964252) | anything practice | - |
+
+### osu!taiko
+
+Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91aedb2923e167f79fe92)!
```
Comment (Walavouchey) [inline wiki/Contests/VMC/2/en.md:107]:
```suggestion
Detailed scores, judge comments, and final placements are available [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91aedb2923e167f79fe92).
```

File: wiki/Contests/VMC/2/en.md [hunk 3]
```diff
@@ -88,6 +88,50 @@ Please be sure to support [iroha(sasaki)](https://osu.ppy.sh/beatmaps/artists/39
...
+
+| Placement | User | Anonymised beatmap | Badge prize |
+| :-: | :-- | :-- | :-- |
+| 1st | [uone](https://osu.ppy.sh/users/5321719) | try storm | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
+| 2nd | [Nifty](https://osu.ppy.sh/users/4956097) | naturally if | - |
+| 3rd | [-Kazuha](https://osu.ppy.sh/users/29978316) | birthday slight | - |
+
+### osu!catch
+
+Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91b13b2923e167f79feec)!
```
Comment (Walavouchey) [inline wiki/Contests/VMC/2/en.md:117]:
```suggestion
Detailed scores, judge comments, and final placements are available [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91b13b2923e167f79feec).
```

File: wiki/Contests/VMC/2/en.md [hunk 4]
```diff
@@ -88,6 +88,50 @@ Please be sure to support [iroha(sasaki)](https://osu.ppy.sh/beatmaps/artists/39
...
+
+| Placement | User | Anonymised beatmap | Badge prize |
+| :-: | :-- | :-- | :-- |
+| 1st | [Dika312](https://osu.ppy.sh/users/741613) | happily under | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
+| 2nd | [Unlucky_w](https://osu.ppy.sh/users/4820793) | curve blue | - |
+| 3rd | [Cipse](https://osu.ppy.sh/users/7928880) | flow late | - |
+
+### osu!mania
+
+Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91a82b2923e167f79fe20)!
```
Comment (Walavouchey) [inline wiki/Contests/VMC/2/en.md:127]:
```suggestion
Detailed scores, judge comments, and final placements are available [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91a82b2923e167f79fe20).
```

File: news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+| 3rd | [Meijiro McQueen](https://osu.ppy.sh/users/11555612) | having mental | - |
+| 4th | [celerih](https://osu.ppy.sh/users/4696296) | president summer | - |
+| 5th | [Nijika Ijichi](https://osu.ppy.sh/users/10964252) | anything practice | - |
+
+#### Commentary
+
+See the maps with commentary presented by [Noffy](https://osu.ppy.sh/users/1541323), [-White](https://osu.ppy.sh/users/16276548), and [Halfslashed](https://osu.ppy.sh/users/4598899). Find out which maps were each judge's favourites, receive valuable insight into judge comments, and other interesting bits of information!
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/inFJsE586t0" frameborder="0" allowfullscreen></iframe>
```
Comment (Walavouchey) [inline news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md:42]: all these videos need to be unprivated

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "Vocaloid Mapping Contest 2 Results"
```
Comment (Walavouchey) [inline news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md:3]: ci error

```suggestion
title: Vocaloid Mapping Contest 2 Results
```

File: news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+This contest's submissions were immediately sent to the eyes of our discerning [judges](/wiki/Contests/VMC/2#judges) after the mapping phase. Despite the relatively short mapping phase, the participants did a great job, and their passion for the music is clear.
+
+[See the full results here.](https://mappersguild.com/contests/results?contest=65d91885b2923e167f79fd44)
+
+#### Summary
+
+| Placement | User | Anonymised beatmap | Badge prize |
+| :-: | :-- | :-- | :-- |
+| 1st | [Nuolong](https://osu.ppy.sh/users/4732455) | world language | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
```
Comment (Walavouchey) [inline news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md:31]: image

this badge is a bit too big for the table cell, should use the 1x version instead

**Comment by pishifat** [comment]
instead of listing the anonymous map names, could you instead do something like `[song](maplink)`? the anon names are meaningless

File: news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md (hunk 7 above)
Comment (Walavouchey) [reply news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md:31]: never.. mind?

image

File: news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md (hunk 6 above)
Comment (osuWhite) [reply news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md]: applied

File: wiki/Contests/VMC/2/en.md [hunk 8]
```diff
@@ -22,7 +22,7 @@ This second iteration of the VMC featured a new VOCALOID Featured Artist, suppor
 | Judging | 2024-03-08 to 2024-04-05 |
 | Results livestream | 2024-04-07 @ 18:00 UTC |
 
-The results are streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. Further details regarding the stream will be provided in this article, the news post, and in the Discord server.
+The results were streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. You can view the recordings for all of the contests on the [*Vocaloid Appreciation Team* YouTube channel](https://www.youtube.com/@VocaloidAppreciationTeam).
```
Comment (Walavouchey) [inline wiki/Contests/VMC/2/en.md:25]: avoiding pronouns in the wiki

```suggestion
The results were streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. Recordings for all of the contests are available on the [*Vocaloid Appreciation Team* YouTube channel](https://www.youtube.com/@VocaloidAppreciationTeam).
```

File: news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md (hunk 7 above)
Comment (osuWhite) [reply news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md]: wala applied

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**